src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const names = document.querySelectorAll('li');
+
+function sortList(personList) {
+  return personList.sort((a, b) => b.salary - a.salary);
+}
+
+function getEmployees(people) {
+  return [...people].map(person => {
```

Comment:
the `map` method returns a new array, a copy is redundant here
```suggestion
  return people.map(person => {
```